src/mongocxx/collection.hpp

```diff
@@ -19,6 +19,7 @@
 #include <memory>
 #include <string>
 
+#include <bsoncxx/builder/basic/array.hpp>
```

Comment:
Fixing up the headers for index_view revealed that this header was missing from collection.hpp